paginate the events when there are too many

this is not urgent for now, but this may become needed if the number of events grows too much (both for UX and latency reason)

We had to deactivate what was done with pagination so far because we observed (when our own agenda had enough events to actually require several pages) those errors: 

```
[2022-10-07T16:26:40.326970+02:00] SlackAPI.ERROR: API call failed in views_publish. [] []
[2022-10-07T16:26:40.494645+02:00] SlackAPI.ERROR: raw response: {"ok":false,"error":"invalid_arguments","response_metadata":{"messages":["[ERROR] failed to match all allowed schemas [json-pointer:\/view]","[ERROR] must provide an object [json-pointer:\/view\/blocks\/64]"]}} [] []
```

The payload looked like this:

```
{
	"user_id": "U023NPAU6BE",
	"view": {
		"type": "home",
		"blocks": [{
				"type": "section",
				"text": {
					"type": "mrkdwn",
					"text": "*L\u00e9gende:* :desktop_computer: \u00e9v\u00e9nement en visioconf\u00e9rence; :white_circle: \u00e9v\u00e9nement\/action accessible aux nouveaux b\u00e9n\u00e9voles; :large_blue_circle: \u00e9v\u00e9nement\/action \u00e0 venir observer (si vous voulez monter en comp\u00e9tence); :large_purple_circle: \u00e9v\u00e9nement\/action n\u00e9cessitant d'\u00eatre bien pr\u00e9par\u00e9 (ex: formation comme intervenant).\n\n :warning: Si jamais vous vous \u00eates inscrit.e \u00e0 une action mais que vous ne pouvez finalement plus participer, n'oubliez pas de vous *d\u00e9sinscrire*, et *si c'est la veille de l'action*, merci d'en informer aussi <@UQ7NC358X> au 06 18 76 17 94."
				}
			}, {
				"type": "header",
				"text": {
					"type": "plain_text",
					"text": "\u00c9v\u00e8nements \u00e0 venir"
				}
			}, {
				"type": "section",
				"block_id": "filter_section",
				"text": {
					"type": "mrkdwn",
					"text": "Choisissez vos filtres"
				},
				"accessory": {
					"action_id": "filters_has_changed",
					"type": "multi_static_select",
					"placeholder": {
						"type": "plain_text",
						"text": "Filtres"
					},
					"options": [{
						"text": {
							"type": "plain_text",
							"text": "Mes \u00e9v\u00e8nements"
						},
						"value": "my_events"
					}]
				}
			}, {
				"type": "divider"
			}, {
				"type": "section",
				"text": {
					"type": "mrkdwn",
					"text": "event description"
				}
			}, {
				"type": "actions",
				"block_id": "81c82ac2-1b00-4520-a7bc-3721a401339f.ics",
				"elements": [{
					"type": "button",
					"action_id": "getin",
					"text": {
						"type": "plain_text",
						"text": "Je  viens !",
						"emoji": true
					},
					"style": "primary",
					"value": "approve"
				}, {
					"type": "button",
					"action_id": "more",
					"text": {
						"type": "plain_text",
						"text": "Plus d'informations",
						"emoji": true
					},
					"value": "more"
				}]
			}, {
				"type": "divider"
			},
			[{
				"type": "actions",
				"elements": [{
					"type": "button",
					"text": {
						"type": "plain_text",
						"emoji": true,
						"text": "Page 1"
					},
					"action_id": "page-selection-1",
					"value": "1",
					"style": "primary"
				}]
			}, {
				"type": "divider"
			}], {
				"type": "section",
				"text": {
					"type": "mrkdwn",
					"text": "En cas du bug, veuillez contacter: #c-outils-numeriques"
				}
			}
		]
	}
}
```

--- a/slackEvents.php
+++ b/slackEvents.php
@@ -60,10 +60,11 @@ class SlackEvents {
         $buttons[] = $button;
       }
       
-      array_push($blocks, array([
-            "type" => "actions",
-            "elements" => $buttons
-          ], ["type" => "divider"]));
+      array_push($blocks, array(
+          "type" => "actions",
+          "elements" => $buttons));
+
+      array_push($blocks, array("type" => "divider"));
     }
 
     protected function render_event($parsed_event, $description=false, $with_attendees=true) {
@@ -225,9 +226,7 @@ class SlackEvents {
             array_unshift($blocks, $header_block, $filter_block, ["type"=> "divider"]);
         }
 
-        // Commented out because it "all of a sudden" crashed the app. More details on https://github.com/Zero-Waste-Paris/slack-agenda-app/issues/59#issuecomment-1272378237
-        // (we should either investigate the issue and fix it, or perhaps, eventually, delete all this... :-/
-        //$this->add_page_buttons($blocks, $nof_pages);
+        $this->add_page_buttons($blocks, $nof_pages);
 
         // nothing to show
         if(count($events) === 0) {

